src/credentials/CHIPCertFromX509.cpp

```diff
@@ -693,5 +694,40 @@ DLL_EXPORT CHIP_ERROR ConvertX509CertToChipCert(const uint8_t * x509Cert, uint32
     return err;
 }
 
+CHIP_ERROR ConvertX509CertsToChipCertArray(const ByteSpan & x509Cert1, const ByteSpan & x509Cert2, uint8_t * chipCertArrayBuf,
+                                           uint32_t chipCertArrayBufSize, uint32_t & chipCertBufLen)
+{
+
+    TLVWriter writer;
+    writer.Init(chipCertArrayBuf, chipCertArrayBufSize);
+
+    TLVType outerContainer;
+    //    ReturnErrorOnFailure(writer.StartContainer(ProfileTag(Protocols::OpCredentials::Id.ToTLVProfileId(),
+    //    kTag_ChipCertificateArray), kTLVType_Array, outerContainer));
```

Comment:
There is an issue in the spec I think.

CASE says:

```
. The responder SHALL encode the following items as a <<ref_sigma-2-struct, sigma-2-tbsdata>> with an anonymous tag:
.. `responderNOC` as a <<ref_oce-chip-cert-array, chip-certificate-array>> with an anonymous tag
```

But the spec for cert array indicates an mandatory profile tag

```
==== CHIP Certificate Array

A set of one or more CHIP certificates can be encoded as an array of CHIP certificates. Note that this array MAY be empty.

----
chip-certificate-array [operational-credentials:2] => ARRAY OF chip-certificate
----
```

I think for now using anonymous is fine.